awslabs/aws-lambda-powertools-typescript
# build(layers): aligned aws cdk versions in `layer-publisher` to main workspace

## Description of your changes

As described in #1227, due to a version mismatch between the `aws-cdk-lib` dependencies in the `layer-publisher` and main workspace the integration tests were failing.

The linked issue contains a more exhaustive description of the issue, but the TL;DR; is that if the dependency version used in the two workspaces is not aligned, the integration tests for the layer publisher module will fail.

This PR realigns the version as a stop-gap solution. A proper long term fix is tracked in #1226 and will be proposed as a separate PR in the near future.

### How to verify this change

See result of integration test run here (successful): https://github.com/awslabs/aws-lambda-powertools-typescript/actions/runs/3893918816

### Related issues, RFCs



**Issue number:** #1227

### PR status

***Is this ready for review?:*** YES  
***Is it a breaking change?:*** NO

## Checklist


### Breaking change checklist


---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
